### PR TITLE
[Model Update]: Transmission Pass 2.0.0

### LINKED ISSUE #647: [Model Update]: Transmission Pass
## Update Reason
Update to be conform with the DPP and some new attributes

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [ ] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by catroest**
## Description

 -->

Closes #647 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
@agg3fe ready for review

**Comment by agg3fe** [comment]
> @agg3fe ready for review

Release notes missing :)

File: io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+   samm:characteristic :PerformancePower ;
+   samm:exampleValue "16"^^xsd:nonNegativeInteger .
+
+:TimeMeasurement a samm-c:Measurement ;
+   samm:preferredName "Time Measurement"@en ;
+   samm:description "Time in seconds."@en ;
+   samm:dataType xsd:nonNegativeInteger ;
+   samm-c:unit unit:secondUnitOfTime .
+
+:Characteristic2 a samm-c:List ;
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl:400]: dummy Characteristic? please remove it.

File: io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+:Capacity a samm-c:Quantifiable ;
+   samm:preferredName "Capacity Characteristic"@en ;
+   samm:description "The oil capacity in dm³."@en ;
+   samm:dataType xsd:float ;
+   samm-c:unit unit:cubicDecimetre .
+
+:ElectricalMachineCharacteristic a samm:Characteristic ;
+   samm:preferredName "Electrical Machine Characteristic"@en ;
+   samm:description "Characterisitc for the electrical machine."@en ;
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl:291]: typo `Characterisitc`

File: io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+@prefix ext-passport: <urn:samm:io.catenax.generic.digital_product_passport:4.0.0#> .
+
+:TransmissionPass a samm:Aspect ;
+   samm:preferredName "Transmission Pass"@en ;
+   samm:description "The transmission passport corresponds to the digital product passport information required by the proposed Ecodesign Regulation (ESPR-2022) and describes the data that is collected and available during the lifespan of a transmission."@en ;
+   samm:properties ( :productSpecificParameters :productUnspecificParameters ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:productSpecificParameters a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl:40]: Property and the referenced Characteristic should not have the same name

File: io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+   samm:description "A list of the used transmission damper technology (depending on the transmission type). For example NW 200 TTD and NW 200 ZDW"@en ;
+   samm:characteristic :StringList ;
+   samm:exampleValue "NW 200 TTD" .
+
+:speedResistance a samm:Property ;
+   samm:preferredName "Speed Resistance"@en ;
+   samm:description "The permissible maximum input speed in the respective gear depending on the transmission type."@en ;
+   samm:characteristic :SpeedResistanceList .
+
+:standardGearRatio a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl:99]: Property and the referenced Characteristic should not have the same name

File: io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+   samm:characteristic :GearTrait ;
+   samm:exampleValue "1" .
+
+:speed a samm:Property ;
+   samm:preferredName "Speed"@en ;
+   samm:description "The permissible maximum input speed in the respective gear (depending on the transmission type) in 1/min."@en ;
+   samm:characteristic :RevolutionsPerMinute ;
+   samm:exampleValue "7800"^^xsd:nonNegativeInteger .
+
+:ratio a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/2.0.0/TransmissionPass.ttl:230]: Property and the referenced Characteristic should not have the same name

File: io.catenax.transmission.transmission_pass/RELEASE_NOTES.md [hunk 6]
```diff
@@ -3,6 +3,14 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [1.0.0] - 2024-03-4
```
Comment (agg3fe) [inline io.catenax.transmission.transmission_pass/RELEASE_NOTES.md:6]: version 2.0.0 required

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**